adminapi/src/main/java/io/minio/admin/MinioAdminClient.java

```diff
@@ -613,6 +622,225 @@ public Message getServerInfo() throws IOException, NoSuchAlgorithmException, Inv
+    if (newName != null && !newName.isEmpty()) {
+      serviceAccount.put("newName", newName);
+    }
+    if (newDescription != null && !newDescription.isEmpty()) {
+      serviceAccount.put("newDescription", newDescription);
+    }
+    if (newExpiration != null) {
+      serviceAccount.put("newExpiration", newExpiration.format(Time.EXPIRATION_DATE_FORMAT));
+    }
+
+    Credentials creds = getCredentials();
+    try (Response response =
+        execute(
+            Method.POST,
+            Command.UPDATE_SERVICE_ACCOUNT,
+            ImmutableMultimap.of("accessKey", accessKey),
+            Crypto.encrypt(creds.secretKey(), OBJECT_MAPPER.writeValueAsBytes(serviceAccount)))) {}
+  }
+
+  /**
+   * Deletes a service account by it's access key
+   *
+   * @param accessKey Access Key.
+   * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
+   * @throws InvalidKeyException thrown to indicate missing of HMAC SHA-256 library.
+   * @throws IOException thrown to indicate I/O error on MinIO REST operation.
+   */
+  public void deleteServiceAccount(@Nonnull String accessKey)
+      throws NoSuchAlgorithmException, InvalidKeyException, IOException {
+    if (accessKey == null || accessKey.isEmpty()) {
+      throw new IllegalArgumentException("access key must be provided");
+    }
+
+    try (Response response =
+        execute(
+            Method.DELETE,
+            Command.DELETE_SERVICE_ACCOUNT,
+            ImmutableMultimap.of("accessKey", accessKey),
+            null)) {}
+  }
+
+  /**
+   * Obtains a list of minio service account by user name.
+   *
+   * @param userName user name.
+   * @return List of minio service account.
+   * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
+   * @throws InvalidKeyException thrown to indicate missing of HMAC SHA-256 library.
+   * @throws IOException thrown to indicate I/O error on MinIO REST operation.
+   * @throws InvalidCipherTextException thrown to indicate data cannot be encrypted/decrypted.
+   */
+  public ListServiceAccountResp listServiceAccount(@Nonnull String userName)
+      throws NoSuchAlgorithmException, InvalidKeyException, IOException,
+          InvalidCipherTextException {
+    if (userName == null || userName.isEmpty()) {
+      throw new IllegalArgumentException("user name must be provided");
+    }
+
+    try (Response response =
+        execute(
+            Method.GET,
+            Command.LIST_SERVICE_ACCOUNTS,
+            ImmutableMultimap.of("user", userName),
+            null)) {
+      Credentials creds = getCredentials();
+      byte[] jsonData = Crypto.decrypt(creds.secretKey(), response.body().bytes());
+      return OBJECT_MAPPER.readValue(jsonData, ListServiceAccountResp.class);
+    }
+  }
+
+  /**
+   * Obtains service account info for a specified MinIO user.
+   *
+   * @param accessKey Access Key.
+   * @return Service account info for the specified accessKey.
```

Comment:
Fix all `@return` like `   * @return {@link StatObjectResponse} - Populated object information and metadata.` in the PR